eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -226,4 +244,34 @@ public int transpile(final XML input, final Path target) throws IOException {
             this.generatedDir.toPath()
         ).save();
     }
+
+    /**
+     * Remove transpiled files per EO.
+     * @param src The eo path
+     * @return Count of removed files
+     */
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
```

Comment:
@Karina5005 this suppression is redundant, I believe

Reply:
fixed